src/App.css

```diff
@@ -1,6 +1,4 @@
-body {
-  background-color: #f7f7f7;
-}
+body {}
```

Comment:
scrolling beyond first and last element will show white areas that are different from the header/footer elements